cmd/containerd-shim-runc-v2/task/service.go

```diff
@@ -231,6 +237,9 @@ func (s *service) preStart(c *runc.Container) (handleStarted func(*runc.Containe
 				Container: c,
 				Process:   p,
 			})
+			if init {
+				s.execable[c] = true
```

Comment:
Can we fix this commit to initialize the map (see https://github.com/containerd/containerd/pull/10634#discussion_r1731941581)?  Working intermediate commits helps `git bisect` in the future.

Reply:
Can do!